client/state/sites/actions.js

```diff
@@ -90,6 +91,7 @@ export function requestSites() {
 				site_activity: 'active',
 				fields: SITE_REQUEST_FIELDS,
 				options: SITE_REQUEST_OPTIONS,
+				filters: config.isEnabled( 'jetpack-cloud' ) ? 'atomic,jetpack' : '',
```

Comment:
Instead of checking if jetpack-cloud is enabled why not pass in the sites_filter config value?

Reply:
yeah, that seems like a more easily configurable approach!